### ISSUE TITLE
git doesn't install perl

### ISSUE THREAD

**Issue by jsumners** [voidlinux/void-packages#3296]
From a clean installation of Void:

```bash
$ xbps-install -y git
$ git clone somerepo
$ cd somerepo
$ git submodule init
/usr/libexec/git-core/git-submodule: 168: /usr/libexec/git-core/git-submodule: /usr/bin/perl: not found
```

**Comment by chneukirchen** [comment]
Install `git-perl` or even `git-all`.

**Comment by dominikh** [comment]
I don't think that's a satisfactory answer. Shouldn't `git-submodule` be in `git-perl` then? As it stands, `git` has a runtime dependency that isn't tracked by xbps.

**Comment by jsumners** [comment]
I completely agree. It doesn't make any sense to install a program that won't work because a dependency wasn't installed. That's basically the point of a package manager.

**Comment by Gottox** [comment]
So... Remove git-perl completely and make perl a dep of git?

It isn't nice, but it'll ease development and avoid this kind of bug in the future.

**Comment by dominikh** [comment]
Couldn't we move /usr/libexec/git-core/git-submodule to git-perl
instead? We already have git-add--interactive (git add -i) in
git-perl.

I agree that having git-perl at all is rather uncommon and might
confuse newcomers, but surely there is a reason we have git-perl in
the first place?

On Mon, Jan 04, 2016 at 01:29:00AM -0800, Enno Boland wrote:
> So... Remove git-perl completely and make perl a dep of git?
> 
> It isn't nice, but it'll ease development and avoid this kind of bug in the future.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/voidlinux/void-packages/issues/3296#issuecomment-168620822

-- 
Dominik Honnef

**Comment by Gottox** [comment]
True. We can discuss removing git-perl later. Rather fix the actual problem :+1:

**Comment by xtraeme** [comment]
git 2.7 already rewrote some parts of submodule in C, just wait for the next version before changing our srcpkg.

**Comment by Gottox** [comment]
I disagree. This is a real issue, there's a simple fix (move submodule to git-perl), this should be fixed. We can move it back once submodule won't use perl anymore.

**Comment by dominikh** [comment]
Fixed in 5f8da98258ae0ab651cbb72b11a6dd468d401cce, which moved git-submodule to git-perl. If anyone wants to track other git/perl related issues, open new ones.
